bin/smack.py

```diff
@@ -376,6 +401,8 @@ def verify_bpl(args):
     command += ["/cex:%s" % args.max_violations]
     command += ["/maxStaticLoopBound:%d" % args.loop_limit]
     command += ["/recursionBound:%d" % args.unroll]
+    if args.context_switches:
```

Comment:
So this is fixed in my next commit, however, now, I'm wondering why it should be a problem that Corral would have `k` set to 2 by default.  For single-threaded input programs, is there any difference when `k == 1` vs `k == 2`, since there are no opportunities for context switches, anyway?